tests/tools.py

```diff
@@ -119,45 +119,8 @@ def running_under_travis():
     return 'TRAVIS' in os.environ and os.environ['TRAVIS'] == 'true'
 
 
-def get_osutil_for_travis():
-    distro_name = os.environ['_system_name'].lower()
-    distro_version = os.environ['_system_version']
-
-    if distro_name == "ubuntu" and distro_version == "14.04":
-        return Ubuntu14OSUtil()
-
-    if distro_name == "ubuntu" and distro_version == "16.04":
-        return Ubuntu16OSUtil()
-
-
-def mock_get_osutil(*args):
-    # It's a known issue that calling platform.linux_distribution() in Travis will result in the wrong info.
-    # See https://github.com/travis-ci/travis-ci/issues/2755
-    # When running in Travis, use manual distro resolution that relies on environment variables.
-    if running_under_travis():
-        return get_osutil_for_travis()
-    else:
-        return _get_osutil(*args)
-
-
 def are_cgroups_enabled():
-    # We use a function decorator to check if cgroups are enabled in multiple tests, which at some point calls
-    # get_osutil. The global mock for that function doesn't get executed before the function decorators are imported,
-    # so we need to specifically mock it beforehand.
-    mock__get_osutil = patch("azurelinuxagent.common.osutil.factory._get_osutil", mock_get_osutil)
```

Comment:
From what I got from the code, this was the only place where this method was actually needed. Is that understanding correct?